### PR TITLE
chore: Namananand/ins 2103 landing page hackoctober

### PR DESCRIPTION
**Pull request by iamnamananand996**
Because

- landing page hackoctober

This commit

- landing page hackoctober

### CODE REVIEW

File: src/pages/landing.tsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+    // But in this way we can not use useElementDimension hook which is
+    // troublesome. Even We use ref and get element dimension in the useEffect
+    // that calculate the line stat. The time activeIndex change didn't mean
+    // it's the time DOM had been updated and every component had been drew.
+
+    window.onbeforeunload = function () {
+      window.scrollTo(0, 0);
+    };
+  }, []);
```
Comment (EiffelFly) [inline src/pages/landing.tsx:123]: @iamnamananand996 You can safely remove this part~

File: src/pages/landing.tsx
Comment (EiffelFly) [inline src/pages/landing.tsx:1]: @iamnamananand996 Let remove the old page

File: src/pages/index.tsx [hunk 2]
```diff
@@ -185,76 +79,983 @@ const HomePage: FC<HomePageProps> & {
...
+                      <Button
+                        variant="primary"
+                        className="!rounded-none !rounded-r-sm"
+                      >
+                        Subscribe
+                      </Button>
+                    </div>
+                  </div>
+                </div>
+              </div>
```
Comment (EiffelFly) [inline src/pages/index.tsx:978]: @iamnamananand996 The subscription button doesn't have the corresponding function

**Review by EiffelFly — COMMENTED** [review]
@iamnamananand996 

- Let remove old pages
- We should add the functionality of subscription newsletter

**Comment by iamnamananand996** [comment]
Hi @EiffelFly , updated the suggested changes

**Review by EiffelFly — COMMENTED** [review]
@iamnamananand996 Please check these comments

**Comment by iamnamananand996** [comment]
Hi @EiffelFly , updated the changes

**Review by EiffelFly — APPROVED** [review]
LGTM

**Pull request merged**